src/main/flight/mixer.c

```diff
@@ -353,7 +397,9 @@ static void applyMixToMotors(float motorMix[MAX_SUPPORTED_MOTORS], motorMixer_t
     for (int i = 0; i < mixerRuntime.motorCount; i++) {
         float motorOutput = motorOutputMixSign * motorMix[i] + throttle * activeMixer[i].throttle;
 #ifdef USE_THRUST_LINEARIZATION
+    if (!mixerConfig()->rpm_limiter) {
```

Comment:
`if (!mixerConfig()->rpm_limiter)` should be replaced with  if (!RPM_LIMIT_ACTIVE)
where `RPM_LIMIT_ACTIVE` is from my previous comment:
```
#ifdef USE_RPM_LIMITER
    #define RPM_LIMIT_ACTIVE mixerConfig()->rpm_limiter
#else
    #define RPM_LIMIT_ACTIVE false
#endif
```